src/popup/components/ErrorKeyRevoked.tsx

```diff
@@ -35,16 +36,35 @@ export const ErrorKeyRevoked = ({
     return (
       <AnimatePresence mode="sync">
         <MainScreen
-          requestReconnect={() => setScreen('reconnect')}
           disconnectWallet={disconnectWallet}
           onDisconnect={onDisconnect}
+          setScreen={setScreen}
+        />
+      </AnimatePresence>
+    );
+  } else if (screen === 'consent-reconnect') {
+    return (
+      <AnimatePresence mode="sync">
+        <AutoKeyAddConsent
+          onAccept={async () => {
+            try {
+              await reconnectWallet({ autoKeyAddConsent: true });
+              onReconnect?.();
+              clearScreen();
```

Comment:
If `onReconnect` reloads page, we'll never `clearScreen`

```suggestion
              clearScreen();
              onReconnect?.();
```